### ISSUE TITLE
Works except the drone

### ISSUE THREAD

**Issue by Hacker05964** [N74NK/MLBBTOOL#2]
This tool blow my mind that it really works except for Drone.Please can you fix it and can you add Diamond Hack for this...It's mind blowing

**Comment by BLACKHAT-POPEYE** [comment]
Yes its work except for Drone, but how to disable the background?

**Comment by maulaaskaputri** [comment]
Send me
